app/views/upscan_csv_file_upload.scala.html

```diff
@@ -36,29 +44,36 @@
 }
 
 @pageTitle = @{
-  uploadFileHeader + " " + messages("ers.title.postfix").toString
+  uploadFileHeader + " " + messages("ers.title.postfix")
 }
 
-@ers_main(title = pageTitle.toString, scripts) {
-
+@govuk_wrapper(title = pageTitle, Some(scripts)) {
     @reference(requestObject.getPageTitle)
-    <p class="font-xsmall"><a href="@routes.CheckCsvFilesController.checkCsvFilesPage" class="link-back">@messages("ers.back")</a></p>
 
-    <div id="errors" class="flash error-summary error-summary--show validation-summary visibility" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
-        <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
-        @messages("validation.summary.heading")
+    @govukBackLink(BackLink(
+        href = routes.CheckCsvFilesController.checkCsvFilesPage.url,
+    ))
+
+    <div class="govuk-error-summary govuk-!-display-none" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+        <h2 class="govuk-error-summary__title" id="error-summary-title">
+            @messages("validation.summary.heading")
         </h2>
-        <ul class="js-error-summary-messages">
-            <li><a href="#file-uploader" data-focuses="file-uploader">@messages("csv_file_upload.summary_error")</a></li>
-        </ul>
+        <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+            </ul>
+        </div>
     </div>
 
-    <label class="form-label" id="fileName" for="fileToUpload" data-file-name="@messages(ersUtil.getPageElement(schemeId, ersUtil.PAGE_CHECK_CSV_FILE,fileId+".file_name"))">
-      @messages("csv_file_upload.upload_your_file", messages(ersUtil.getPageElement(schemeId, ersUtil.PAGE_CHECK_CSV_FILE,fileId+".file_name")))
-    </label>
-    @upscan_file_upload_form(".csv", upscanInitiateResponse.formFields, upscanInitiateResponse.postTarget, uploadFileHeader)
-    <div class="panel-indent">
-        <p>@messages("file_upload.delay.csv")</p>
-    </div>
+    @upscan_file_upload_form(
```

Comment:
on this page, the error seems to persist (even after choosing a correct file, the error is still displayed), and there is no error message by the file upload (only the summary) - I had the same issue with ers-checking